test/e2e/baremetal_test.go

```diff
@@ -62,4 +63,22 @@ func TestBaremetalDefaults(t *testing.T) {
 	if want := "StorageNotConfigured"; conds.Degraded.Reason() != want {
 		t.Errorf("degraded reason: got %q, want %q", conds.Degraded.Reason(), want)
 	}
+
+	clusterOperator := framework.MustEnsureClusterOperatorStatusIsSet(t, client)
+	for _, cond := range clusterOperator.Status.Conditions {
+		switch cond.Type {
```

Comment:
This check is pointless. `MustEnsureClusterOperatorStatusIsSet` waiting for `available=True`, `progressing=False`, `Degraded=False`.

Reply:
@legionus perhaps it makes sense to modify `MustEnsureClusterOperatorStatusIsSet` to just verify that the ClusterOperator object exists?